dashboard/config/levels/custom/spritelab/allthethings_fish_tank.level

```diff
@@ -41,7 +41,7 @@
+    "start_animations": "{\r\n  \"orderedKeys\": [\r\n    \"64d0d5ae-6038-4f02-85ba-ca5db6a03d27\",\r\n    \"8aea38dd-f919-47f0-82be-18b74c4ae08c\",\r\n    \"163dc7ea-619b-41d5-9285-f0bfbd6600c4\"\r\n  ],\r\n  \"propsByKey\": {\r\n    \"64d0d5ae-6038-4f02-85ba-ca5db6a03d27\": {\r\n      \"name\": \"fish\",\r\n      \"sourceUrl\": \"https://studio.code.org/api/v1/animation-library/spritelab/CwVno2kER.r_tECMOL4D4YL.lcaFNv7h/category_animals/fish.png\",\r\n      \"frameSize\": {\r\n        \"x\": 100,\r\n        \"y\": 100\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"tHiJgwp46yeRprLgyImU1QXKXh17vdzv\"\r\n    },\r\n    \"8aea38dd-f919-47f0-82be-18b74c4ae08c\": {\r\n      \"name\": \"tumbleweed\",\r\n      \"sourceUrl\": \"https://studio.code.org/api/v1/animation-library/level_animations/souuCV88zXZv.NWAGwJGGzb8ykjCYAGd/tumbleweed1.png\",\r\n      \"frameSize\": {\r\n        \"x\": 100,\r\n        \"y\": 93\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"7HdhwfAAYeLpryFaAeDJIzpYTQi7gfpk\"\r\n    },\r\n    \"163dc7ea-619b-41d5-9285-f0bfbd6600c4\": {\r\n      \"name\": \"cactus\",\r\n      \"sourceUrl\": \"https://studio.code.org/api/v1/animation-library/spritelab/fvP_YDSp32ON.B1R69F.ha4Et2iu.U5W/category_obstacles/cactus.png\",\r\n      \"frameSize\": {\r\n        \"x\": 73,\r\n        \"y\": 100\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"WE5jFaTZzJaTbT.sFemB6i.YLmERTMQh\"\r\n    }\r\n  }\r\n}",
```

Comment:
A little tricky to tell from this diff... what is the intended change?

Reply:
Reverting each sourceUrl to use full studio.code.org host instead of relative